DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -531,21 +531,21 @@ public DMASTProcStatement ProcStatement()
 
                 return new DMASTProcStatementExpression(loc, expression);
             } else {
-                DMASTProcStatement procStatement = ProcVarDeclaration();
-                if (procStatement == null) procStatement = Return();
-                if (procStatement == null) procStatement = Break();
-                if (procStatement == null) procStatement = Continue();
-                if (procStatement == null) procStatement = Goto();
-                if (procStatement == null) procStatement = Del();
+                DMASTProcStatement procStatement = Return();
```

Comment:
```suggestion
                // These are sorted by frequency, except If() is moved to the end because it's really slow (relatively)
                DMASTProcStatement procStatement = Return();
```